editor/src/components/inspector/controls/color-picker.tsx

```diff
@@ -37,6 +44,17 @@ export const ColorPicker: React.FunctionComponent<React.PropsWithChildren<ColorP
   portalTarget,
   ...props
 }) => {
+  const onSubmitValue = props.onSubmitValue
+  const dispatchEyeDropper = React.useCallback(() => {
```

Comment:
I think dispatch is misleading here, I think it should be something like onEyeDropper, onClickEyeDropper or handleEyeDropper

```suggestion
  const onClickEyeDropper = React.useCallback(() => {
```

Reply:
I applied your change manually in https://github.com/concrete-utopia/utopia/pull/3115/commits/34d554fd5b886f09b40e6dda45f117673941acc9